### ISSUE TITLE
🛹 Feature: Form Group with Error

### ISSUE THREAD

**Issue by bastiW** [public-ui/kolibri#5923]
> **Note**
> - [x] Ich achte beim Erstellen des Issues darauf, dass ich **keine** fachlichen und unternehmensinternen Details preisgebe.

---

# Featureanfrage

I would like a form or input group to display different forms and one error message for all checkboxes. 

## Bezieht sich Ihre Feature-Anfrage auf ein Problem? Bitte beschreiben.
When we have several checkboxes there is no posiblity to just display one error. 

## Beschreiben Sie die gewünschte Lösung

There should be a form group. 
The Form or input group can have
- Inputs
- Checkboxes

The form group can show an error for the whole form group

## Beschreiben Sie Alternativen, die Sie in Betracht gezogen haben
Develop custom for project.

**Comment by laske185** [comment]
The container to wrap one error message over multiple input elements could look like this:

```tsx
const hasError = true;
const errorMsg = "Error";
return <>
  <fieldset>
    <legend>Telefonnummer</legend>
    {hasError && 
    (
      <kol-alert _alert _type="error" _label={errorMsg}></kol-alert>
    )}
    <kol-input-text _label="Länderkennung" _hide-label _error={errorMsg} _hide-error /></kol-input-text>
    <kol-input-text _label="Vorwahl" _hide-label _error={errorMsg} _hide-error /></kol-input-text>
    <kol-input-text _label="Nummer" _hide-label _error={errorMsg} _hide-error /></kol-input-text>
  </fieldset>
</>;
```

**Comment by laske185** [comment]
To-do: Beispiel in der react-sample-app anlegen.

Anschließend kann evaluiert werden, ob es sich lohnt, hierfür eine eigene Komponente anzulegen.
